ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/util/BlockBlobSidecarsTrackersPoolImpl.java

```diff
@@ -294,32 +262,14 @@ private void publishRecoveredBlobSidecar(final BlobSidecar blobSidecar) {
     blobSidecarGossipPublisher.apply(blobSidecar).ifExceptionGetsHereRaiseABug();
   }
 
-  private void publishRecoveredDataColumnSidecars(
-      final BlockBlobSidecarsTracker blockBlobSidecarsTracker) {
-    final SignedBeaconBlock signedBeaconBlock = blockBlobSidecarsTracker.getBlock().orElseThrow();
-    final MiscHelpersFulu miscHelpersFulu =
-        MiscHelpersFulu.required(spec.atSlot(signedBeaconBlock.getSlot()).miscHelpers());
-    final List<DataColumnSidecar> dataColumnSidecars =
-        miscHelpersFulu.constructDataColumnSidecars(
-            signedBeaconBlock,
-            blockBlobSidecarsTracker.getBlobSidecars().values().stream()
-                .map(BlobSidecar::getBlob)
-                .toList(),
-            kzg);
-    LOG.info(
-        "Publishing {} data column sidecars for {}",
-        dataColumnSidecars.size(),
-        blockBlobSidecarsTracker.getSlotAndBlockRoot());
-    dataColumnSidecarPublisher.accept(dataColumnSidecars);
-  }
-
   private void countBlobSidecar(final RemoteOrigin origin) {
     switch (origin) {
       case RPC -> poolStatsCounters.labels(COUNTER_SIDECAR_TYPE, COUNTER_RPC_SUBTYPE).inc();
       case GOSSIP -> poolStatsCounters.labels(COUNTER_SIDECAR_TYPE, COUNTER_GOSSIP_SUBTYPE).inc();
       case LOCAL_EL ->
           poolStatsCounters.labels(COUNTER_SIDECAR_TYPE, COUNTER_LOCAL_EL_SUBTYPE).inc();
-      case LOCAL_PROPOSAL ->
+      // FIXME: I stink
```

Comment:
Did you forget this or is this gonna be fixed in another PR? :)

Reply:
Nice catch. Fixed!